po/de.po

```diff
@@ -0,0 +1,1112 @@
+msgid "Copy backup to a safe place"
+msgstr "Sicherung an einen sicheren Ort kopieren"
+
+#: public/views/backup.html
+msgid "Copy this text as it is to a safe place (notepad or email)"
+msgstr "Diesen Text an einem sichern Ort einfügen (Notepad oder E-Mail)"
+
+#: public/views/backup.html
+msgid "Copy to clipboard"
+msgstr "In die Zwischenablage kopieren"
+
+#: src/js/controllers/walletHome.js
+msgid "Could not accept payment. Check you connection and try again"
+msgstr ""
+"Zahlung konnte nicht angenommen werden. Bitte die Verbindung  überprüfen und "
+"erneut versuchen"
+
+#: src/js/controllers/walletHome.js
+msgid "Could not broadcast payment. Check you connection and try again"
+msgstr ""
+"Zahlung kann nicht übermittelt werden. Bitte die Verbindung prüfen und "
+"erneut probieren"
+
+#: src/js/controllers/walletHome.js
+msgid ""
+"Could not connect wallet service. Check your Internet connexion and your "
+"wallet service configuration."
+msgstr ""
+"Kann keine Verbindung zum Wallet Service aufgebaut werden. Bitte die "
+"Internetverbindung und die Wallet Service Einstellung überprüfen."
+
+#: src/js/controllers/walletHome.js
+msgid "Could not create address. Check you connection and try again"
+msgstr ""
+"Es kann keine Adresse erzeugt werden. Bitte Verbindung prüfen und erneut "
+"versuchen"
+
+#: src/js/controllers/walletHome.js
+msgid "Could not create payment proposal"
+msgstr "Es kann kein Zahlungsvorschlag erzeugt werden"
+
+#: src/js/services/profileService.js
+msgid "Could not create using the specified extended private key"
+msgstr "Erzeugung mit erweiterten privaten Schlüssel nicht möglich"
+
+#: src/js/controllers/import.js
+msgid "Could not decrypt file, check your password"
+msgstr "Datei kann nicht entschlüsselt werden, bitte das Passwort überprüfen"
+
+#: src/js/controllers/walletHome.js
+msgid "Could not delete payment proposal. Check you connection and try again"
+msgstr ""
+"Die Zahlungsanweisung kann nicht gelöscht werden. Bitte Verbindung prüfen "
+"und erneut probieren"
+
+#: src/js/controllers/walletHome.js
+msgid "Could not fetch payment information"
+msgstr "Zahlungsinformationen können nicht abgerufen werden"
+
+#: public/views/walletHome.html
+msgid "Could not fetch transaction history"
+msgstr "Transaktionshistorie kann nicht abgerufen werden"
+
+#: src/js/services/profileService.js
+msgid "Could not import. Check input file and password"
+msgstr "Import nicht möglich. Bitte Datei und Passwort überprüfen"
+
+#: src/js/services/profileService.js
+msgid "Could not join using the specified extended private key"
+msgstr ""
+"Teilnahme mit spezifizierten erweiterten privaten Schlüssel nicht möglich"
+
+#: src/js/controllers/join.js
+msgid "Could not join wallet:"
+msgstr "Beteiligung am Wallet nicht möglich"
+
+#: src/js/controllers/walletHome.js
+msgid "Could not reject payment. Check you connection and try again"
+msgstr ""
+"Zahlung konnte nicht abgelehnt werden. Bitte Verbindung prüfen und erneut "
```

Comment:
"konnte" vs "kann" inconsistent translation of "could" across these three messages